features/support/log_message.rb

```diff
@@ -0,0 +1,62 @@
+class LogMessage
```

Comment:
```suggestion
class ChromeBrowserLog
```

This is more precise in the general context of the Smokey repo.